vscode-antimony/src/syntax/antimony.tmLanguage.json

```diff
@@ -119,7 +119,7 @@
         "keywords": {
             "patterns": [
                 {
-                    "match": "\\b(?:identity)\\b",
+                    "match": "\\b(?:identity|biological_entity_is|hasPart|part|isPartOf|parthood|isVersionOf|hypernym|hasVersion|version|isHomologTo|homolog|isDescribedBy|description|isEncodedBy|encoder|encodes|encodement|occursIn|container|hasProperty|property|isPropertyOf|propertyBearer|hasTaxon|taxon|sboTerm)\\b",

```

Comment:
This is pretty awkward.  Can you use ANNOT_KEYWORD?  Maybe not in a .json file...

Reply:
> 

Could you be a little more specific about how should I format this?

Reply:
Sadly, I don't know how formatting things in .json files works.  My hope was that you could have one element that was x|y|z|p|d|q and then refer to that element in this 'match' line.  But perhaps that's impossible?

If it is, and if it's possible to add line returns without messing things up, I'd do something like:
     [...]  identity|
            biological_entity_is|
            hasPart|
[...]

If not, as-is works, I guess!

Reply:
Sounds like we're leaving it as-is!